src/engine/Input/Pointer.ts

```diff
@@ -89,6 +92,7 @@ export class PointerEvent extends GameEvent<any> {
                public button: PointerButton,
                public ev: any) {
       super();
+      this.pos = new Vector(x, y);
```

Comment:
There should be 1 source of truth for x,y.

x,y can be set independently from pos.

Either let's make x/y getters that ref pos, or make pos a getter that references x/y